spec.md

```diff
@@ -54,7 +58,7 @@ Several terms are used frequently in this document and warrant basic definitions
 - **Pull**: the act of downloading Blobs and Manifests from a Registry
 - **Blob**: the binary form of content that is stored by a Registry, addressable by a Digest
 - **Manifest**: a JSON document which defines an Artifact. Manifests are defined under the OCI Image Spec <sup>[apdx-2](#appendix)</sup>
-- **Config**: a section in the Manifest (and associated Blob) which contains Artifact metadata
+- **Config**: a blob referenced in the Manifest which contains Artifact metadata
```

Comment:
This is also described in [image-spec](https://github.com/opencontainers/image-spec/blob/v1.0.1/config.md) if you want to link to that (similar to how manifest links to apdx-2).

Reply:
Thanks again, just pushed changes addressing this and the above